docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -195,6 +195,7 @@ For the `s3` backend, the following additional properties are supported in the `
 
 For the `gcs` backend, the following additional properties are supported in the `config` attribute:
 
+- `enable_bucket_policy_only`: When `true`, the GCS bucket that is created to store the state will be configured to use uniform bucket-level access.
```

Comment:
We should shift this under `skip_bucket_versioning` so its consistent with above.

Reply:
Got it